dev-vcs/tortoisehg/tortoisehg-5.3.1.ebuild

```diff
@@ -69,8 +50,11 @@ python_install_all() {
 	domenu contrib/thg.desktop
 
 	# Remove file that collides with >=mercurial-4.0 (bug #599266).
-	rm "${ED}"/usr/$(get_libdir)/${EPYTHON}/site-packages/hgext3rd/__init__.py* \
+	rm "${ED}"/usr/$(get_libdir)/${EPYTHON}/site-packages/hgext3rd/__init__.* \
```

Comment:
This is just so wrong. Don't guess paths when `python_get_sitedir` gives you correct paths. Also, it would be preferable to remove this prior to installing if possible (e.g. in `prepare`, or in `python_install` from BUILD_DIR/lib/... before calling the eclass function).

Reply:
Thanks for advice.